api/Models/Developer.cs

```diff
@@ -0,0 +1,19 @@
+
+namespace api.Models
+{
+    public class Developer
+    {
+        public string Id { get; set; } = string.Empty;
+
+        public string Name { get; set; } = string.Empty;
+        public string? RomanizedName { get; set; } = string.Empty;
+
+        public Employee[]? Credits { get; set; }
+
+        public DateTime? YearsActiveStart { get; set; }
+        public DateTime? YearsActiveEnd { get; set; }
+
+        public Game[]? GamesCreated { get; set; }
```

Comment:
I would change this array types to be List<Game> (or IEnumeraable<Game>) as it buys you a little more with stuff like null bounds checking out of the box.
```suggestion
        public List<Game> GamesCreated { get; set; } = new List<Game>();
```

Reply:
thx